guides/source/active_record_callbacks.md

```diff
@@ -31,80 +40,95 @@ irb> @baby = Baby.create
 Congratulations!
 ```
 
-As you will see, there are many life cycle events and you can choose to hook into any of these either before, after, or even around them.
+As you will see, there are many life cycle events and you can choose to hook
+into any of these either before, after, or even around them.
 
-Callbacks Overview
+Callback Registration
 ------------------
 
-Callbacks are methods that get called at certain moments of an object's life cycle. With callbacks it is possible to write code that will run whenever an Active Record object is created, saved, updated, deleted, validated, or loaded from the database.
+To use the available callbacks, you need to implement and register them.
+Implementation can be done in a multitude of ways like using ordinary methods,
+blocks and procs, or defining custom callback objects using callback classes.
+Let's go through each of these implementation techniques.
 
-### Callback Registration
-
-In order to use the available callbacks, you need to register them. You can implement the callbacks as ordinary methods and use a macro-style class method to register them as callbacks:
+You can implement the callbacks as a **macro-style method that calls an ordinary
+method** for registration.
 
 ```ruby
 class User < ApplicationRecord
-  validates :login, :email, presence: true
```

Comment:
In the Callback Registration Section we discuss the different ways to register and implement a callback, hence I've updated the example to be the same across all the different ways so that they're comparable and easier to follow. 

In addition, a `login` attribute/column didn't make complete sense to me so I used something more common like a `username` column.